src/pal/src/memory/heap.cpp

```diff
@@ -26,7 +26,7 @@ Revision History:
 #include "pal/handlemgr.hpp"
 #include "pal/corunix.hpp"
 #include <errno.h>
-
+#define HEAP_SIZEINFO_SIZE (2 * sizeof(DWORD))
```

Comment:
Why is it different from the expression used instead of this define before (max(sizeof(void*),sizeof(double));)